tests/ciscotest.rb

```diff
@@ -176,6 +177,16 @@ def skip_nexus_i2_image?
       Utils.nexus_i2_image
   end
 
+  def system_image
+    @image ||= Platform.system_image
+  end
+
+  def defect?(pattern, msg)
+    return false unless system_image.match(Regexp.new(pattern))
+    puts "\n#{self.class}##{caller[0][/`.*'/][1..-2]} -> NOOP <- [#{msg}]"
```

Comment:
The logger message is not quite as pretty but okay. :-)
The NOOP/pass message was an ask from Mike. We agreed to assert on "wont-fix" bugs only. Defects that have a chance of being fixed should still fail.